bubuku/cli.py

```diff
@@ -227,5 +227,30 @@ def show_stats():
         _print_table(table)
 
 
+@cli.group(name='validate', help='Validates internal structures of kafka/zk')
+def validate():
+    pass
+
+
+@validate.command('replication', help='Checks zk structure for correct replica list')
```

Comment:
@adyach maybe the help should be clearer, then. I would suggest `Returns all partitions whose ISR size differs from the replication factor`.